scrapi/harvesters/addis_ababa.py

```diff
@@ -6,6 +6,27 @@
 from __future__ import unicode_literals
 
 from scrapi.base import OAIHarvester
+from scrapi.base import helpers
+
+
+def format_uris_addis_ababa(*args):
+    identifiers = helpers.gather_identifiers(args)
+    provider_uris, object_uris = helpers.seperate_provider_object_uris(identifiers)
+
+    for arg in args:
+        if arg and 'http://hdl.handle.net/123456789/' in arg[0]:
+            doc_id = arg[0].replace('http://hdl.handle.net/123456789/', '')
+            canonical_uri = 'http://etd.aau.edu.et/handle/123456789/' + doc_id
```

Comment:
This is sort of a minor quibble, but `str.format` is usually the preferred way to format strings: 
```python
canonical_uri = 'http://etd.aau.edu.et/handle/123456789/{}'.format(doc_id)
```